core/src/surplus.ts

```diff
@@ -54,11 +54,23 @@ const getAuctionState = async (network: string, earliestEndDate: Date, greatestB
     return 'just-started';
 };
 
+const _getContractAuctionDuration = async (network: string) => {
```

Comment:
As suggested, it would be better to call this using `surplus` in its name
```suggestion
const _fetchSurplusAuctionsDurationInSeconds = async (network: string) => {
```